.env.example

```diff
@@ -5,4 +5,6 @@ DBNAME=yourDBName
 JWT_SECRET=anySecretYouWantHere
 EXPIRING_TIME=600
 NODE_ENV=development
-SALT_ROUNDS=10
\ No newline at end of file
+SALT_ROUNDS=16
+ITERATIONS=100000
```

Comment:
What is salt_rounds and iterations?

Reply:
`Salting is simply the addition of a unique, random string of characters known only to the site to each password before it is hashed, typically this “salt” is placed in front of each password`
I've taken this definition from [here](https://www.theguardian.com/technology/2016/dec/15/passwords-hacking-hashing-salting-sha-2)

Reply:
Good.